src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -92,7 +87,7 @@ def create_llm_inputs(
         num_of_output_prompts: int = DEFAULT_NUM_OF_OUTPUT_PROMPTS,
         add_model_name: bool = False,
         add_stream: bool = False,
-        tokenizer: LlamaTokenizerFast = None,
+        tokenizer: AutoTokenizer = None,
```

Comment:
should we set to our default tokenizer instead of `None`?

Reply:
Probably

Reply:
yea good call. Done